Jenkinsfile

```diff
@@ -65,6 +65,17 @@ elifePipeline {
                     return m
                 })
             }
+
+            stage 'Trigger grobid and grobid trainer tag update', {
+                def grobidTag = allGrobidTags.last()
+                def grobidTrainerTag = "${grobidTag}-${commit}"
+                build   job: 'dependencies/dependencies-sciencebeam-trainer-delft-update-grobid-and-trainer',
```

Comment:
I assume this runs from https://alfred.elifesciences.org/job/libraries/job/library-sciencebeam-trainer-grobid/, so you need:
```
'../dependencies/dependencies-sciencebeam-trainer-delft-update-grobid-and-trainer'
```
as a relative path.

Reply:
just a general unrelated question - what criteria determines which pipeline remain in root path and which goes into `image`, `dependencies`, `libraries`, `jobs` . .

Reply:
Probably only `test-*` and `prod-*` pipelines should stay at the top level, as we select them from the [build monitor](https://alfred.elifesciences.org/view/Build%20Monitor%20(projects)/). The assumption is that if a library fails is less important than a real deployment failing.